test/jdk/javax/management/remote/mandatory/connection/DefaultAgentFilterTest.java

```diff
@@ -164,9 +169,24 @@ public void close() throws Exception {
     }
 
     private static final String TEST_APP_NAME = "TestApp";
+    private static final int FREE_PORT_ATTEMPTS = 10;
 
     private static void testDefaultAgent(String propertyFile) throws Exception {
```

Comment:
This is a good refactoring change as it also removes some broken logic in the current retry strategy